### ISSUE TITLE
Bitfinex Orders Get Stuck

### ISSUE THREAD

**Issue by renkcub** [michaelgrosner/tribeca#127]
Works fine on Gdax but orders get stuck on Bitfinex. Closing of orders manually results in new orders being placed. 

Any ideas for fix or anyone else experiencing?

**Comment by Camille92** [comment]
Stupid question but did you turn off tribeca ?

Otherwise it's normal that orders are updated.

**Comment by renkcub** [comment]
No, tribeca remains on. They are somehow getting stuck. I can check logging for more details. I'm working on a bandaid that would involved just cancelling any orders once every few minutes in case it gets stuck.

**Comment by michaelgrosner** [comment]
Do ALL orders stay open? Or just a few seemingly randomly stay open?

**Comment by renkcub** [comment]
Just a few randomly, "stopping" the bot until I cancel the rogue order.
Going to see if I can find the issue in logging.
